util/travis_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+TRAVIS_COMMIT_MESSAGE="${TRAVIS_COMMIT_MESSAGE:-none}"
+TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE:-HEAD~1..HEAD}"
+
+# test force push
+#TRAVIS_COMMIT_RANGE="c287f1bfc5c8...81f62atc4c1d"
+
+NUM_IMPACTING_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE} | grep -Ev '^(docs/)' | wc -l)
```

Comment:
Just to make sure, this would only trigger if only files in `/docs` are altered. Anything else will cause a full compile, correct? 

I believe that's the case, but wanted to make sure.

Reply:
Correct, the logic is "list all changes on branch, then filter out anything in the docs folder" if the number is non zero, there is work to do.

Reply:
Okay, just making sure.  The scripting stuff is one area that I'm not so great in.